.github/workflows/docs-remove-preview.yml

```diff
@@ -0,0 +1,56 @@
+name: Docs Remove Preview
+
+env:
+  CARGO_TERM_COLOR: always
+
+on:
+  workflow_run:
+    workflows: [PR Closed]
+    types:
+      - completed
+
+jobs:
+  teardown:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'pull_request'
+    steps:
+      - name: Get PR number
+        uses: actions/github-script@v6
+        id: get-pr
+        continue-on-error: true
+        with:
+          script: |
+            let login = context.payload.workflow_run.head_repository.owner.login;
+            let branch = context.payload.workflow_run.head_branch;
+            let head = `${login}:${branch}`;
+            let prs = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: "closed",
+              head,
+            });
+            if (prs.data.length > 0) {
+              core.setOutput("prnumber", prs.data[0].number);
+            } else {
+              core.setFailed("Could not find PR");
+            }
+      
+      - name: Teardown preview
+        if: steps.get-pr.outcome == 'success'
+        continue-on-error: true
+        uses: adrianjost/actions-surge.sh-teardown@v1.0.3
+        with:
+          regex: pr-${{ steps.get-pr.outputs.prnumber }}--ruma-docs.surge.sh
```

Comment:
`steps.get-pr.outputs` looks really weird since identifiers in most programming languages can't have `-` in them. Can we use `_` in job IDs?